tests/features/actions/visit.js

```diff
@@ -0,0 +1,12 @@
+import nightmare from 'nightmare'
+import url from 'url'
+
+import { options, url as BASE_URL } from '../browser'
+
+export default function (path = '', query = {}) {
+  const location = url.resolve(BASE_URL, path)
+  return (
+    nightmare(options)
+      .goto(location)
```

Comment:
no its just a browser action. The way i've configured the feature tests is to destructure the browser actions as much as possible from the actual tests. Doing it this way allows the feature tests to be composed of many different browser actions (each spawning their own nightmare instance...thus avoiding the issues noted [here](https://github.com/rosshinkley/nightmare-examples/blob/master/docs/common-pitfalls/async-operations-loops.md)

Because we are using the async/await syntax, we can decompose all the steps into their own actions, each returning a promise, then we can await them, then do the assertions on the output from the resolved (or rejected) promise. 

This also allows for actions to be used across multiple feature tests so we don't write duplicate browser steps.